website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -52,165 +52,8 @@ unit_tests:
 
 </file>
 
```

Comment:
Are the Unit Tests properties configured in the project.yml only? If so, we should say that and maybe we should add unit-tests to the configuration file found here: https://docs-getdbt-com-git-unit-testing-ff-dbt-labs.vercel.app/reference/dbt_project.yml

or are they added to any .yml file under a tests or unit tests key? If so, then something like this might be helpful:

> Unit tests properties can be declared in .yml files under a tests key.

Reply:
Adding the "before you begin" highlights from the /docs file, which includes file location and format